lib/models/user.go

```diff
@@ -0,0 +1,13 @@
+package models
+
+import "gorm.io/gorm"
+
+type User struct {
+	Base              Base   `gorm:"embedded;"`
+	Email             string `gorm:"unique"`
```

Comment:
```suggestion
	Email             string `gorm:"unique;"`
```

It is not mandatory, but as we added it for `gorm:"embedded;` on the previous line, it can help with consistency 👍